docs/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "preprocess": "yarn workspace @noir-lang/acvm_js build && ./scripts/codegen_nargo_reference.sh && yarn node ./scripts/preprocess/index.js",
-    "dev": "yarn preprocess && ENV=dev docusaurus start",
+    "dev": "yarn preprocess && yarn version::stables && ENV=dev docusaurus start",
```

Comment:
I'd like to avoid running `yarn version::stables` as part of `yarn dev` as it's a fairly slow step which doesn't benefit from incremental compilation. We should update the docs to mention that this should be run before `yarn dev`.